core/server/models/post.js

```diff
@@ -232,6 +233,18 @@ Post = ghostBookshelf.Model.extend({
                 authors.forEach(author => author.emitChange('attached', options));
             });
         });
+        
+        model.related('posts_meta').once('detaching', function onDetached(collection, postsMeta) {
+            model.related('posts_meta').once('detached', function onDetached(detachedCollection, response, options) {
+                postsMeta.emitChange('detached', options);
```

Comment:
Do we have a use-case for this event? We are only emitting them for Webhooks and some special communication with the "frontend". Not sure it makes sense to have them unless there's a specific need? :thinking:

Reply:
🤔 Hmm yeah didn't have any usecase for them in mind, just added them for consistency with rest of relations. I'll remove this and the `emitChange` method added in `postsMeta` table 👍

Reply:
@gargol Done 🔥